src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java

```diff
@@ -1008,7 +1008,7 @@ else if(model instanceof AcogReticle) {
                                 }, 250, 0))
 
                         .build())
-                .withSpawnEntityDamage(5.4f)
+                .withSpawnEntityDamage(6.4f)
```

Comment:
please revert this change (I will do extended research on balance of weapons at a later date)